ex_1_v2.py

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Tue Jun  5 15:29:27 2018
+
+@author: amyhaddad
+"""
+
+
+name=input("What is your name?")
```

Comment:
The style preference is to have spaces around the assignment to the variable, so:

```
name = input("What is your name?")
```

Reply:
To get the terminal to print a space between the prompt (What is your name?) and the user's input, add a space after the question mark.